app/presenters/collection_presenter.rb

```diff
@@ -2,6 +2,8 @@
 
 # Presents a collection
 class CollectionPresenter < FormPresenter
+  include LinkHelper
```

Comment:
Can we drop this `include` since it's also included in the parent class?

Reply:
I forgot to hit save ...